site/app/Resources/views/base.html.twig

```diff
@@ -11,17 +11,12 @@
   <link rel="icon" type="image/png" href="{{ asset('favicon.png') }}" sizes="32x32">
 
   {% block stylesheets %}
-    {% stylesheets filter="scssphp,cssrewrite" output="css/app.css"
-        "assets/scss/librecores.scss"
-        "assets/scss/bootstrap-librecores.scss"
-        "assets/css/bootstrap-social.css"
-        "assets/css/font-awesome.min.css"
-        "assets/css/librecores.css"
-    %}
-        <link rel="stylesheet" href="{{ asset_url }}" />
-    {% endstylesheets %}
-  {% endblock %}
+    {# 'app' must match the first argument to addEntry() in webpack.config.js #}
+    {{ encore_entry_link_tags('app') }}
 
+    <!-- Renders a link tag (if your module requires any CSS)
+         <link rel="stylesheet" href="/build/app.css"> -->
```

Comment:
Is that needed for CSS to get included?

Reply:
I don't think so. We have included all of them in the `app` argument

Reply:
And it is automatically included from there. I checked `dev tools
`